contrib/ansible/roles/skydive_common/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+- package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - libselinux-python
+    - python2-ruamel-yaml
+  when: ansible_os_family == "RedHat"
+
+- include: binary.yml
+  when: deployment_mode == "binary"
+
+- include: package.yml
+  when: deployment_mode == "package"
+
+- stat:
+    path: /etc/skydive/skydive.yml
+  register: configfile
+
+- name: ensures skydive configuration folder exists
+  file: path=/etc/skydive state=directory
+
+- get_url:
+    url: https://raw.githubusercontent.com/skydive-project/skydive/{{ skydive_release }}/etc/skydive.yml.default
+    dest: /etc/skydive/skydive.yml
+    mode: 0644
+  when: not configfile.stat.exists
```

Comment:
You can just use get_url with "stat" and the conditional, since if "force" is by default to "no", so it will not replace the current file if it exist.

Reply:
Done